tests/neg-custom-args/explicit-nulls/byname-nullables.check

```diff
@@ -1,7 +1,7 @@
 -- [E007] Type Mismatch Error: tests/neg-custom-args/explicit-nulls/byname-nullables.scala:19:24 -----------------------
 19 |    if x != null then f(x)   // error: f is call-by-name
    |                        ^
-   |                        Found:    (x : String | Null)
+   |                        Found:    x.type /*String | Null*/
```

Comment:
Clearly the old message is better. Non-experts would not know what x.type even is!